template/App.js

```diff
@@ -6,44 +6,85 @@
  * @flow
  */
 
-import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
+import React, {Fragment} from 'react';
+import {StyleSheet, ScrollView, View, Text, StatusBar} from 'react-native';
 
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android:
-    'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
+import Header from 'react-native/Libraries/NewAppScreen/components/Header';
+import LearnMoreLinks from 'react-native/Libraries/NewAppScreen/components/LearnMoreLinks';
+import Colors from 'react-native/Libraries/NewAppScreen/components/Colors';
+import DebugInstructions from 'react-native/Libraries/NewAppScreen/components/DebugInstructions';
+import ReloadInstructions from 'react-native/Libraries/NewAppScreen/components/ReloadInstructions';
+
+const Section = ({children}) => (
```

Comment:
We added this component in the original PR to add this functionality (see https://github.com/facebook/react-native/pull/24687#discussion_r280485176), but I'm starting to question whether this is a Good Component™ for us here or not.

Given that it doesn't handle anything but applying the style here, I'm not sure if it's worth the extra overhead or not? What thoughts do y'all have?

Reply:
Kill it.